custom_components/audiconnect/dashboard.py

```diff
@@ -390,178 +393,302 @@ def create_instruments():
         TripData(attr="longterm_reset", name="LongTerm Trip User Reset"),
         Lock(),
         Preheater(),
-        Sensor(attr="model", name="Model", icon="mdi:car-info", unit=None),
-        Sensor(attr="mileage", name="Mileage", icon="mdi:speedometer", unit="km"),
+        Sensor(
+            attr="model",
+            name="Model",
+            icon="mdi:car-info",
+            unit=None,
+            state_class=None,
+            device_class=None,
+        ),
+        Sensor(
+            attr="mileage",
+            name="Mileage",
+            icon="mdi:speedometer",
+            unit="km",
+            state_class="total_increasing",

```

Comment:
I would think we should be importing these up front vs text and comparison. 

For example:

`from homeassistant.components.sensor import (
    SensorDeviceClass,
    SensorEntity,
    SensorEntityDescription,
    SensorStateClass,
)`

followed by: 

`state_class=SensorStateClass.TOTAL_INCREASING`


Examples are shown here: https://github.com/Hyundai-Kia-Connect/kia_uvo/blob/master/custom_components/kia_uvo/sensor.py

Reply:
oh yeah, this is much cleaner. I'll clean it up.